modes/ja/_shared.md

```
@@ -8,217 +8,277 @@
-1. 経験やメトリクスを捏造する
-2. `cv.md` やポートフォリオファイルを変更する
-3. 候補者の代わりに応募を送信する
-4. 生成メッセージで電話番号を共有する
-5. 市場以下の報酬を推奨する
-6. 求人を読まずに PDF を生成する
-7. 企業ジャーゴンや「お役所言葉」を使う
-8. tracker を無視する（評価したすべての求人を記録する）
+Recruiter-side risk mapping、six-second clarity、business-value bullets、ATS reality checks については `modes/heuristics/recruiter-side.md` を読む。
 
-### 常にする
+### Avoid cliché phrases
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)